edge_computing/image-base-install/ibi-edge-image-based-install.adoc

```diff
@@ -0,0 +1,74 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ibi-image-based-install_{context}"]
+= About image-based deployments for {sno} clusters
+include::_attributes/common-attributes.adoc[]
+:context: ibi-edge-image-based-install
+
+toc::[]
+
+When a host preinstalled with {sno} arrives at a remote site, a technician can easily reconfigure and deploy the host in a matter of minutes.
```

Comment:
This doc is about IBI, but, maybe we should be specific here that a host preinstalled with an SNO *installed via IBI*  will reconfigure, to avoid someone thinking that this reconfiguration applies to all SNOs ?